dessia_common/sampling.py

```diff
@@ -93,7 +93,7 @@ def _get_doe(self, method: str = 'fullfact', samples: int = None, lhs_criterion:
             return self._montecarlo_sampling(samples=samples)
         raise NotImplementedError(f"Method '{method}' is not implemented in {self.__class__}._get_doe method.")
 
-    def make_doe(self, method: str = 'fullfact', samples: int = None, lhs_criterion: str = 'center', name: str = ''):
+    def make_doe(self, method: str = 'fullfact', samples: int = 10, lhs_criterion: str = 'center', name: str = ''):
```

Comment:
Delete default 10 values

Reply:
Removed and fixed one mistake.
Do we keep defaults value for method ?